app/__init__.py

```diff
@@ -332,3 +352,26 @@ def get_reviews():
     output.append(spot_data)
   return { 'data': output }
 
+
+# returns count for each rating for individual beach/area ["1"] ["2"] ["3"], etc
+# returns count for total ratings ["total"]
+# returns average rating for beach ["average"]
+@app.route("/review/getsummary")
+def get_summary_reviews():
+  beach_id = request.args.get('beach_id')
```

Comment:
since all this logic is duplicated, can we make a helper function that does this and then call it both in this API endpoint and also in the other one?